src/main/resources/sam.conf

```diff
@@ -177,17 +177,5 @@ azureServices {
   managedAppClientId = ${?AZURE_MANAGED_APP_CLIENT_ID}
   managedAppClientSecret = ${?AZURE_MANAGED_APP_CLIENT_SECRET}
   managedAppTenantId = ${?AZURE_MANAGED_APP_TENANT_ID}
-  managedAppPlans = [
```

Comment:
Wondering, should `managedAppPlans` be in sam.conf but read the values from env-vars? I think BEEs have `independentReleases` set to true, meaning firecloud-develop isn't used: https://github.com/broadinstitute/terra-helmfile/blob/32f075ea593cc234541b9f759930b1bb2e38169a/values/app/sam/bee.yaml.gotmpl#L22

"Live" environments still use firecloud-develop for the time being.

@broadinstitute/identiteam should confirm though, I haven't been in the weeds as much w/ Sam configs.